src/artemis/system_tests/test_fgs_plan.py

```diff
@@ -6,6 +6,11 @@
 
 from artemis.devices.eiger import EigerDetector
```

Comment:
Should: As discussed, these are the same as https://github.com/DiamondLightSource/python-artemis/pull/508/files, I will review that then we can remove the changes here.